contracts/queue/src/contract.rs

```diff
@@ -135,7 +137,7 @@ fn query_sum<S: Storage, A: Api, Q: Querier>(deps: &Extern<S, A, Q>) -> StdResul
     let values: StdResult<Vec<Item>> = deps
         .storage
         .range(None, None, Order::Ascending)?
-        .map(|(_, v)| from_slice(&v))
+        .map(|item| item.and_then(|(_, v)| from_slice(&v)))
```

Comment:
Another nice way to handle this :smile:

Reply:
Yeah, merges errors of `item` and `from_slice`. Nice if you know the Option API by heard. Otherwise ... mäh